packages/eslint-config-khan/a11y.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/no-commonjs */
+const ERROR = "error";
+
+module.exports = {
+    parser: "@typescript-eslint/parser",
+    plugins: ["@khanacademy"],
+    env: {
+        // TODO(csilvers): once we properly use node.js for node
+        // files, get rid of this next line.
+        node: true,
+        browser: true,
+        es6: true,
+        jest: true,
+    },
+    settings: {
+        react: {
+            version: "detect",
```

Comment:
Updated the settings for the react version following what we have in `webapp`: https://github.com/Khan/webapp/blob/81dea05cb7a6534847a43e6b057cef0431f60b8c/services/static/.eslintrc.js#L928-L930